eng/build.ps1

```diff
@@ -121,7 +121,7 @@ foreach ($argument in $PSBoundParameters.Keys)
     "buildtests"        { if ($build -eq $true) { $arguments += " /p:BuildTests=true" } else { $arguments += " -build /p:BuildTests=only" } }
     "test"              { $arguments += " -test" }
     "configuration"     { $configuration = (Get-Culture).TextInfo.ToTitleCase($($PSBoundParameters[$argument])); $arguments += " /p:ConfigurationGroup=$configuration -configuration $configuration" }
-    "framework"         { $arguments += " /p:TargetGroup=$($PSBoundParameters[$argument].ToLowerInvariant())"}
+    "framework"         { if ($PSBoundParameters[$argument].ToLowerInvariant() -eq 'netcoreapp') { $arguments += " /p:TargetGroup=netcoreapp5.0" } else { if ($PSBoundParameters[$argument].ToLowerInvariant() -eq 'netfx') { $arguments += " /p:TargetGroup=net472" } else { $arguments += " /p:TargetGroup=$($PSBoundParameters[$argument].ToLowerInvariant())"}}}
```

Comment:
Hmm, I'm not a fan of hardcoding the mapping here.  Can we handle this differently?

Reply:
This is just a temporary fix until the yaml files are finalized, then i go ahead and delete this. I can add a comment here to remove the stuff

Reply:
Are you delaying the yaml change to avoid git conflicts?

Reply:
yes